zyte_spider_templates/spiders/ecommerce.py

```diff
@@ -150,28 +150,39 @@ class EcommerceCrawlStrategyParam(BaseModel):
                     ),
                 },
                 EcommerceCrawlStrategy.direct_item: {
-                    "title": "Direct URLs to Product",
+                    "title": "Direct URLs",
                     "description": (
-                        "Directly extract products from the provided URLs, "
-                        "without any crawling. To use this strategy, pass "
-                        "individual product URLs to the spider, not the "
-                        "website or product category URLs. Common use cases "
-                        "are product monitoring and batch extraction."
+                        "Directly extract items from the provided URLs, "
+                        "without any crawling. To use this strategy, pass to "
+                        "the spider individual product or product list URLs "
+                        "(in line with the extract spider parameter value). "
+                        "Common use cases are product monitoring and batch "
+                        "extraction."
                     ),
                 },
             },
         },
     )
 
 
-class FollowProductLinksParam(BaseModel):
-    dont_follow_product_links: bool = Field(
-        title="Do not follow product links",
-        description=(
-            "Extract ProductList items from product list pages instead of following "
-            "product links and extracting Product items from them."
-        ),
-        default=False,
+@document_enum
+class EcommerceExtract(str, Enum):
+    product: str = "product"
+    """
+    Product data from a product details page.
```

Comment:
Hm, plural seems a bit more natural here: "product data from product detail pages"; the same for productList